src/crystal/thread_local_value.cr

```diff
@@ -0,0 +1,27 @@
+# :nodoc:
+class Crystal::ThreadLocalValue(T)
```

Comment:
Not sure why a `class` would be necessary, since the underlying store is a `Hash`, thus passed always passed by reference.